assets/css/main.css

```diff
@@ -258,3 +258,301 @@ hr {
+  }
+
+  .header-top li a,
+  .header-top li a i {
+    color: #fff;
+  }
+
+  .header-bottom {
+    display: flex;
+    padding: 0 30px;
+  }
+
+  .header__navLink {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    position: unset;
+    height: 60px;
+    gap: 0;
+    font-weight: var(--fw-400);
+    background: none;
+    padding-top: 0;
+    font-size: 20px;
+    left: unset;
+    right: unset;
+    top: unset;
+    transition: none;
+  }
+
+  .navbar .header__navLink .nav-item {
+    color: var(--sl-black);
+  }
+
+  .nav-button {
+    display: inline-block;
+    padding: 5px;
+  }
+
+  .header__navLink .nav-button a {
+    color: var(--sl-orange);
+  }
+
+  .heading {
+    padding-top: 3rem;
+  }
+
+  .showcase {
+    height: 85vh;
+  }
+
+  .showcase h1 {
+    font-size: 50px;
+    letter-spacing: 5px;
+  }
+
+  .showcase .opener {
+    font-size: 25px;
+    line-height: 30px;
+    font-weight: 600;
+  }
+
+  .showcase .topic {
+    font-size: 20px;
+    line-height: 30px;
+  }
+
+  .showcase .date {
+    font-size: 28px;
+    line-height: 40px;
+  }
+
+  .speakers .btn {
+    display: none;
+  }
+
+  main .programs .btn {
+    display: none;
+  }
+
+  .more {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    border-radius: 50px;
+    width: 40px;
+    height: 40px;
+    background-color: var(--sl-orange);
+    margin: 20px auto;
+  }
+
+  main .programs a:hover {
+    color: var(--sl-orange);
+    font-weight: var(--fw-700);
+  }
+
+  .cards-wrapper {
+    display: grid;
+    grid-template-columns: repeat(5, 1fr);
+    justify-content: center;
+    align-items: center;
+    padding: 3px 130px;
+  }
+
+  .speakers .card-wrapper {
+    display: grid;
+    grid-template-columns: repeat(3, 1fr);
+    align-items: center;
+    gap: 15px;
+    padding: 3px 130px;
+  }
+
+  .programs {
+    padding: 10px 130px;
+  }
+
+  .footer {
+    padding: 0 130px;
+  }
+
+  .sponsors-list img {
+    margin-top: -20px;
+  }
+}
+
+@media screen and (min-width: 1024px) {
+  .header {
+    display: block;
+  }
+
+  .showcase h1 {
+    font-size: 55px;
+    letter-spacing: 5px;
+    margin: 10px 0;
+  }
+
+  .showcase .opener {
+    font-size: 20px;
+    line-height: 30px;
+    font-weight: 600;
+    padding-top: 50px;
+  }
+
+  .showcase .topic {
+    font-size: 20px;
+    line-height: 30px;
+    width: 600px;
+    padding: 10px 1px;
+  }
+
+  .showcase .date {
+    font-size: 25px;
+    line-height: 40px;
+    margin: 10px 0;
+  }
+
+  .showcase .address {
+    font-size: 19px;
+    font-weight: 500;
+  }
+
+  .heading {
+    padding: 2rem;
+  }
+
+  .header .fa-bars {
+    display: none;
+  }
+
+  .header-bottom,
+  .header-top {
+    display: flex;
+    padding: 0 130px;
+    color: var(--sl-black);
+  }
+
+  .header-bottom {
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  .showcase {
+    width: 100vw;
+    height: 100vh;
+    padding: 90px 130px;
+  }
```

Comment:
- [ ] Your page is really amazing, just a small thing you can make it better is when I load the page I see scrolling on the headline section as on the pic. You can fix this issue be updating the padding here to `padding: 90px 130px 0px;` to give it padding-bottom = zero.

![image](https://user-images.githubusercontent.com/35535130/163071719-f1ecb21a-8a9a-426a-9457-88151709a5a3.png)